src/pages/User/User.tsx

```diff
@@ -96,7 +97,10 @@ const User = (): JSX.Element => {
       )}
       <div className={styles.mainColumn}>
         <Routes>
-          <Route path="my-account" element={<AccountComponent panelClassName={styles.panel} panelHeaderClassName={styles.panelHeader} />} />
+          <Route
+            path="my-account"
+            element={<AccountComponent panelClassName={styles.panel} panelHeaderClassName={styles.panelHeader} canUpdateEmail={!integration?.inplayer} />}
```

Comment:
I think it is more scaleable if we inspect the actual controller / service functionality instead of just making everything an if inplayer / cleeng check.  For example, this should be coming from accountController.canUpdateEmail() which will return `accountService.updateEmail !== undefined` (I assume here to split updateUser and updateEmail into 2 methods instead of 1 method with 2 arg types.)

Essentially, we should look at the capabilities of the services we're using, not just the config. It is more semantic, doesn't require inferring how configs and api's work, and will scale nicely as additional functionality or services are added (for example, if we ever add email update functionality to inplayer, we can just add it to the service and everything else will just work without any additional changes)